src/components/TipInput.vue

```diff
@@ -125,22 +112,22 @@ export default {
     };
   },
   computed: {
-    ...mapGetters(['account', 'balance', 'isLoggedIn', 'loading']),
+    ...mapGetters(['account', 'loading']),
     eventPayload() {
       return `${this.tip.id}:${this.show}`;
     },
     deepLink() {
-      const url = new URL(`${process.env.VUE_APP_WALLET_URL}/tip`);
+      const url = new URL(`${process.env.VUE_APP_WALLET_URL}/${this.isRetip ? 're' : ''}tip`);
```

Comment:
This kind of unreadable, I would put that in a separate method that returns the url (endpoint)

Reply:
```suggestion
      const url = new URL(`${process.env.VUE_APP_WALLET_URL}/${this.isRetip ? 'retip' : 'tip'}`);
```
Would you prefer this more? Actually, this is already a method (computed property) that returns the url, don't see a reason to split it into two methods.